iOS/DoraemonKit/Src/Core/Plugin/Performance/MotionMonitor/Data/DoraemonMotionDataSource.m

```diff
@@ -0,0 +1,62 @@
+//
+//  DoraemonMotionDataSource.m
+//  CocoaAsyncSocket
+//
+//  Created by tianYang on 2022/11/4.
+//
+
+#import "DoraemonMotionDataSource.h"
+
+@implementation DoraemonMotionDataSource {
+    dispatch_semaphore_t semaphore;
+}
++ (DoraemonMotionDataSource *)shareInstance{
```

Comment:
`sharedInstance` or `shared` looks like a better name?

Reply:
We use the same naming method as the other singletons in Dokit
<img width="525" alt="image" src="https://user-images.githubusercontent.com/19932964/210086546-3207d2fe-0794-4f42-bfec-f8980659da12.png">